frontend/libs/studio-components/src/components/StudioAvatar/StudioAvatar.module.css

```
@@ -0,0 +1,7 @@
+.avatar {
+  --avatar-size: 40px;
+
+  width: var(--avatar-size);
+  height: var(--avatar-size);
+  border-radius: 100%;
```

Did the calculation based on size not work? There's nothing technically wrong with using 100%, but I believe 50% is mathematically more accurate for creating a circular shape with a border-radius.

```suggestion
  border-radius: 50%;
```

---

40px / 2 is 20px, and therefore it looked weird 😅

---

But 20px is round shape with the size is 40px x 40px 🥳